pegasus/sites.v3/code.org/public/project-ideas.md.partial

```diff
@@ -0,0 +1,370 @@
+---
+title: Project ideas
+video_player: true
+theme: responsive
+---
+<style>
+  table {
+    table-layout: fixed;
+    width: 100%;
+  }
+  td {
+    vertical-align: top;
+    text-align: center;
+  }
+  img {
+    width: 100%;
+  }
+</style>
+
+<a id="top"></a>
+# Project Ideas for K-5 Elementary School Students
+
+[Sprite Lab](https://code.org/educate/spritelab) can be used by anyone but is
+targeted at elementary school students. Programming in a brand new environment
+can be challenging, but you can do it! For more information about full course
+offerings, check out [CS Fundamentals](https://code.org/educate/curriculum/
+elementary-school) to find the right course for your age and skill level.
+
+## [**Sprite Lab**](https://code.org/educate/spritelab)
+<div class="col-33" style="padding-right: 20px;">
+  <img src="/images/spritelab/rikki.jpg">
+</div>
+<div class="col-66">
+  Sprite Lab is a block-based programming environment where you can make
+  simple animations and games with objects and characters that interact with
+  each other. Create an animation or game then share your app!
+</div>
+<div style="clear: both; margin-bottom: 20px"></div>
+<table>
+  <thead>
+    <tr>
+      <th>Sprite Lab Project Ideas</th>
+      <th></th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <h3><strong>Virtual Pet</strong></h3>
+        <p>
+          Drag the food or the toy to the pet and watch the sprites interact.
+        </p>
+        <img src="/images/spritelab/pet.png" style="max-width:50%">
+        <br>
+        <br>
+        <p>
+          <a href="https://studio.code.org/s/coursef-2021/lessons/4/levels/1">
+            Try it out!
+          </a>
+        </p>
+      </td>
+      <td>
+        <h3><strong>Greeting Card</strong></h3>
+        <p>Enter your name and age to get a personalized greeting.</p>
+        <img src="/images/spritelab/birthday.png" style="max-width:50%"/>
+        <br>
+        <br>
+        <p>
+          <a href="https://studio.code.org/s/coursef-2021/lessons/8/levels/2/
+          sublevel/1">
+            Try it out!
+          </a>
+        </p>
+      </td>
+      <td>
+        <h3><strong>Sky Dancer</strong></h3>
+        <p>Collect the coins. Avoid the clouds. The sky's the limit!</p>
+        <img src="/images/spritelab/sky.png" style="max-width:50%"/>
+        <br>
+        <br>
+        <p>
+          <a href="https://studio.code.org/s/coursef-2021/lessons/20/levels/1/
+          sublevel/2">
+            Try it out!
+          </a>
+        </p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+# Project Ideas for Middle and High School Students
+[Web Lab](https://code.org/educate/weblab), [Game Lab](https://code.org/educate/
+gamelab) and [App Lab](https://code.org/educate/applab) can be used by anyone
```

Comment:
Sorry for the late comment - noticed on staging that I think there's a typo here? See screenshot
![image](https://user-images.githubusercontent.com/68714964/140986019-76f2852d-6b8e-4d9a-9676-98ea2ed383c5.png)

Reply:
Great catch! Thanks, Dan! https://github.com/code-dot-org/code-dot-org/pull/43485